.changeset/twelve-cobras-remember.md

```diff
@@ -0,0 +1,5 @@
+---
+'@protocolink/logics': patch
+---
+
+fix: remove stable rate tests
```

Comment:
Wondering which one is better: removing or skipping?

Reply:
Seems AAVE will not reopen the stable borrow in the near future. We will remove the tests.